docker.linuxserver.swag/bin/docker.linuxserver.swag

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+name="letsencrypt"
```

Comment:
Isn't it supposed to be "swag"?

Reply:
Derp

Reply:
Thanks for the heads up
https://github.com/linuxserver/libreelec-addon-repo/pull/84